docs/reference/api/pieces.md

```diff
@@ -1,27 +1,29 @@
 # Piece type REST endpoints
 
-Every [piece type](#TODO) has built in REST end points that share their overall structure in common. The exact document properties returned will depend on the piece type's fields.
+Every [piece type](/reference/glossary.md#piece) has built in REST end points that share their overall structure in common. The exact document properties returned will depend on the piece type's fields.
```

Comment:
"has built in REST end points that share their overall structure in common". I suggest rephrasing. Its not clear what you're trying to say here.

Reply:
I don't know either.